ext/openssl/tests/openssl_x509_parse_basic.phpt

```diff
@@ -4,7 +4,7 @@ openssl_x509_parse() tests
 openssl
 --SKIPIF--
 <?php
-if (OPENSSL_VERSION_NUMBER >= 0x30200000) die('skip For OpenSSL < 3.2');
```

Comment:
The `skip` is not part of the message. So it's saying that this test is for OpenSSL < 3.2.

Reply:
I understand. I read it literally that the "test is skipped for OpenSSL < 3.2". There are a few cases in the codebase with reversed logic. I wonder why this format was chosen. Anyway, I close the PR, sorry for the misunderstanding.

Reply:
No worries. Yeah it's a bit confusing. I'm not sure how to improve it. Some tests add `skip - Message` as a separator, but this will result in a weird message (i.e.  `- Message`). So I would just keep it as is for now.